packages/authorization-process/src/utilities/errorMappers.ts

```diff
@@ -41,3 +42,12 @@ export const deleteClientKeyByIdErrorMapper = (
     .with("clientNotFound", "keyNotFound", () => HTTP_STATUS_NOT_FOUND)
     .with("organizationNotAllowedOnClient", () => HTTP_STATUS_FORBIDDEN)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const removeClientPurposeErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("clientNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with("purposeIdNotFound", () => HTTP_STATUS_BAD_REQUEST)
```

Comment:
I agree with this change, but note that this is not the as-is behaviour, and it could have an impact on QA tests.
Currently the removal of a non-existing purpose silently succeeds.

Reply:
Ok, I reverted the improvement. The additional part is commented and the test is skipped. Can I keep it like so? Or should I delete it entirely?